model/bindings.go

```diff
@@ -72,49 +68,56 @@ func CreateBindingsRancher(pathToYml string) BindingProperty {
 		log.Errorf("Error during Unmarshal  for rancher-compose: %v\n", err)
 	} else {
 		rawConfigRancher, err := preprocess.PreprocessServiceMap(rawConfigRancher)
-		utils.Convert(rawConfigRancher, &rancherComposeMap)
 		if err != nil {
 			fmt.Printf("Error")
 		} else {
-			keys := reflect.ValueOf(rancherComposeMap).MapKeys()
-
-			for _, key := range keys {
-				if key.String() != ".catalog" {
+			for key := range rawConfigRancher {
+				if key != ".catalog" {
 					newServiceBinding := ServiceBinding{}
 
 					newServiceBinding.Labels = mapLabel{}
-					newServiceBinding.Ports = []string{}
+					newServiceBinding.Ports = portArray{}
```

Comment:
These three lines aren't necessary since this already happens when initializing a struct